docs/src/static/robots.njk

```diff
@@ -4,4 +4,3 @@ permalink: robots.txt
 eleventyExcludeFromCollections: true
 ---
 User-agent: *
-Disallow: /
```

Comment:
> **Note**
> Remove this change before merge. needed for URL checkers to parse.

Reply:
This might actually be a mistake. I think we were supposed to remove `Disallow: /` once we went live.

Reply:
yeah i was confused by that.

thought maybe it was so google only indexes the main site, not staging ones or something like that?

Reply:
This prevents crawling of https://latest--docs-eslint.netlify.app, https://docs-eslint.netlify.app, deploy previews and other URLs where `robots.txt` will be in the root, while having no effect on https://eslint.org/ since it isn't in the root there.

Reply:
Ah right. Thanks.